recipes/opcache.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Cookbook Name:: osl-php
+# Recipe:: opcache
+#
+# Copyright 2013, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# if trying to use opcache with an incompatible version of php (default pkg)
+# and using an unsupported version of php. use_ius must be true to install a valid php interpreter.
+
+node.default['osl-php']['php_packages'] << 'opcache'
+
+if node['osl-php']['use_opcache'] && node['php']['version'].to_f < 5.5
+  raise 'Must use PHP >= 5.5 with ius enabled to use Zend Opcache.  Try adding '\
+        "'node.default['osl-php']['use_ius'] = true' and install a proper version of php."
+
+elsif node['osl-php']['use_opcache']
+
+  conf = { 'zend_extension' => 'opcache.so' }
```

Comment:
Why does this need to be included here and not included in the default attribute?

Reply:
Its in attributes/opcache.rb now

Reply:
Is there any reason why those keys can't include the ``opcache.`` prefix in the attribute?

Reply:
I decided to generate the prefix to make things simpler for the person who writes chef code that uses this. Do you think it'd be better to not add the prefix?

Reply:
Yes, I think it would be better not to add the prefix. It just adds complication in case you don't use it in other places.